guides/prow_knative_setup.md

```diff
@@ -31,12 +31,12 @@ All Prow config files for running Prow jobs for Knative projects are under
 1. Make sure that _Knative Prow Robots_ team is an Admin of the repo.
 
 1. Add the new repo to
-   [config_knative.yaml](../config/prow/config_knative.yaml), the meta config
+   [config_knative.yaml](../config/prod/prow/config_knative.yaml), the meta config
    file for generating Prow config and Prow job config. Check the top-level
    section `presubmits:` and `periodics:` for blueprints for what to add. Then
    run `./hack/generate-configs.sh` to regenerate
-   [config/prow/jobs/config.yaml](../config/prow/jobs/config.yaml) and
-   [config/prow/core](../config/prow/core), otherwise the presubmit test in
+   [config/prod/prow/jobs/config.yaml](../config/prod/prow/jobs/config.yaml) and
+   [config/prod/prow/core](../config/prod/prow/core), otherwise the presubmit test in
    test-infra will fail. Create a PR with the changes. Once it's merged the
    configs will be automatically updated by a postsubmit job.
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
   [config/prod/prow/core](../config/prod/prow/core), otherwise the presubmit
   test in test-infra will fail. Create a PR with the changes. Once it's merged
   the configs will be automatically updated by a postsubmit job.
```